llvm/test/CodeGen/AArch64/early-ifcvt-likely-predictable.mir

```diff
@@ -33,35 +33,27 @@ body:             |
   ; CHECK-NEXT:   [[COPY3:%[0-9]+]]:gpr64common = COPY $x0
   ; CHECK-NEXT: {{  $}}
   ; CHECK-NEXT: bb.1:
-  ; CHECK-NEXT:   successors: %bb.3(0x30000000), %bb.2(0x50000000)
+  ; CHECK-NEXT:   successors: %bb.1(0x80000000)
   ; CHECK-NEXT: {{  $}}
-  ; CHECK-NEXT:   [[LDRBBui:%[0-9]+]]:gpr32 = LDRBBui [[COPY3]], 0 :: (load (s8))
```

Comment:
This is a load from an invariant address, which is exactly what the logic should detect. I agree that the `all_of` should be used, but then the predicate probably needs adjusting too